src/main/java/teammates/sqllogic/core/UsersLogic.java

```diff
@@ -205,6 +212,74 @@ public List<User> getAllUsersByGoogleId(String googleId) {
         return usersDb.getAllUsersByGoogleId(googleId);
     }
 
+    /**
+     * Common requirement between reset user by googleId function.
+     * Refer to {@link #resetInstructorGoogleId(String, String, String)} or
+     * {@link #resetStudentGoogleId(String, String, String)}.
+     */
+    private void deleteAccountCascadeAll(String googleId) {
+        if (usersDb.getAllInstructorsByGoogleId(googleId).isEmpty()
+                && usersDb.getAllStudentsByGoogleId(googleId).isEmpty()) {
+            accountsLogic.deleteAccountCascade(googleId);
+        }
+    }
+
+    /**
+     * Resets the googleId associated with the instructor.
+     */
+    public void resetInstructorGoogleId(String email, String courseId, String googleId)
+            throws EntityDoesNotExistException {
+        assert email != null;
+        assert courseId != null;
+        assert googleId != null;
+
+        Instructor instructor = getInstructorForEmail(courseId, email);
+
+        if (instructor == null) {
+            // Is there a better message?
```

Comment:
Questions should not be in production code. Ugly and gives a bad impression, please remove.

Reply:
Sorry, this question was meant for you during review. Will remove the comments 👍🏻